src/router.js

```diff
@@ -147,7 +147,7 @@ export class Router {
     return createRootedPath(path, this.baseUrl, this.history._hasPushState); 
   }
 
-  createNavModel(config) {
+  createNavModel(config:NavModel):NavModel {
```

Comment:
`config` is a route config here, which we'll need an interface for. In the meantime, `Object` would be better.